_posts/2018-06-27-lead-dev-london-2018.md

```diff
@@ -0,0 +1,161 @@
+Share understanding of your work in the team, the whole context is important.
+This can be done by writing good commit messages (also the why not just the what), but also by documenting decisions and the reasons for it, and keeping your documentation up-to-date.
+Taking ownership helps people handle change.
+
+Support and grow individuals. Use every piece of work to help someone grow and start with individual needs.
+Effective delegation can be accomplished by being clear and explicit, explain what's happening and why.
+Also set the scope and boundaries, what's the problem that they're trying to solve, who's involved and when should they bring questions to you.
+Grow the next generation of leaders by teaching people to do your job and make yourself dispensable. 
+
+Show your team where they fit into the wider organisation and how they relate to its goals.
+A wider view helps people adapt when things change.
+Don't over-insulate your team because when something gets through your shield, it will be very disruptive.
+Grow people for a resilient team and organisation, and encourage your team to show off their work, to be proud of it.
+
+Make sure that your manager understands how much work it takes from everyone to build a team that works well and the time it takes to become familiar with the domain they're working in.
+Ask them for support to build a sustainable team and show why all your team's work matters.
+Good communication about change is vital.
+
+> "When stressful things happen, it's easy to default back to not communicating. But that's when it's even more important to empower people during tough times." [@jenny_duckett](https://twitter.com/jenny_duckett){:target="_blank"} [#LeadDevLondon](https://twitter.com/hashtag/LeadDevLondon?src=hash){:target="_blank"}
+
+
+## 'The hardest scaling challenge of all - yourself' by [Christian McCarrick](https://twitter.com/cmccarrick)
+
+<span class="image left"><img class="p-image" alt="Christian McCarrick" src="/img/2018-06-27-Lead-Dev-London-2018/christian-mccarrick.jpg"></span>
+
+**Communication**
+- Be an effective communicator: as you grow, you'll get better at getting your point across to larger and larger groups.
+- Communicate up to manager, across peers and down to your team.
+- Personal branding and self-promotion are important: have confidence and show the value of your work.
+- Create a safe environment by genuinely praising other people and giving honest feedback.
+- Seek out for negative feedback and ask specific questions for better feedback.
+- The most important communication skill to learn is how to say "no" without feeling guilty.
+
+**Prioritization & Time management**
+- If you don't prioritize your life, someone else will.
+- The Eisenhower Matrix: How to make decisions on what's urgent and important.
+
+<img class="image fit center" alt="The Eisenhower Matrix" src="/img/2018-06-27-Lead-Dev-London-2018/eisenhower.png">
```

Comment:
De image hier ziet er "fout" uit. Is het mogelijk om de tekst eruit te halen en de kaders in een andere, cleanere image te steken (transparent background bv)